### DISCUSSION TITLE
各位好. 有问题需要解决的请开新贴子. 别往别的帖子底下留言. 避免漏掉的情况出现. 谢谢各位的合作!

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#596] [Announcements]
各位好. 有问题需要解决的请开新贴子. 别往别的帖子底下留言. 因为太多的留言, 避免漏掉的情况出现. 所以最好您自己的问题自己开一个帖子.谢谢各位的合作!

**Comment by skytony** [comment]
救急，微信已续费，没有更新状态啊

**Reply by wangzissss** [reply]
付费邮箱:[891692309@qq.com](mailto:891692309@qq.com) ,一直没有收到25位激活码，请帮忙一下
![QQ图片20240618104635](https://github.com/getlantern/lantern-forum-cn/assets/166413677/682557e3-7c26-42da-9880-bb2bf3bf8848)
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@wangzissss 好的. 已经帮您提交问题上去了. 请放心. 谢谢您支持蓝灯.

**Reply by wangzissss** [reply]
感谢您，估计要多久能有结果？？

**Reply by MaxMKMax** [reply]
@wangzissss 您好. 问题应该已经解决了. 请重新登陆您的账号看看.

**Reply by MaxMKMax** [reply]
@wangzissss 
您好. 这是您的专属激活码.
W3CJJ-DRWFF-442W7-G3XJK-RQ797

**Reply by wangzissss** [reply]
@Mike777Khoo 苹果手机现在无法使用专业版本对吗？？

**Comment by MaxMKMax** [comment]
@skytony 问题应该已经解决了.

**Comment by wangzissss** [comment]
真的非常感谢您，已经登录专业版
还有一个问题想请教您
登录专业版后，谷歌网站等依然打不开，并且网络很慢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月21日(星期五) 凌晨3:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 各位好. 有问题需要解决的请开新贴子. 别往别的帖子底下留言. 避免漏掉的情况出现. 谢谢各位的合作! (Discussion #596)





 
@wangzissss
 您好. 这是您的专属激活码.
 W3CJJ-DRWFF-442W7-G3XJK-RQ797
